js/isItFriday.js

```diff
@@ -0,0 +1,6 @@
+function isItFriday(date) {
+    let day = date.getDay();
+    return (day === 5);
```

Comment:
Can we avoid hard coded values in-line? Change the 5 to a variable?

Reply:
Added day-of-the-week variables to the latest push!